plugins/find/dialogs/find.js

```diff
@@ -826,6 +829,19 @@
 				}
 			}
 		};
+
+		function createTextNodeWithPreservedSpaces( editor, text ) {
+			var textWithPreservedSpaces = text.replace( consecutiveWhitespaceRegex,
+				function( whitespace ) {
+					var newSpaces = CKEDITOR.tools.array.map( whitespace, function( space, i ) {
```

Comment:
`CKEDITOR.tools.array.map` function takes an array as an argument, not a string. This code takes advantage of strings defining iterator, but it reduces code readability.